smack-integration-test/src/main/java/org/jivesoftware/smack/ChatManagerListenerTest.java

```diff
@@ -0,0 +1,58 @@
+/**
+ *
+ * Copyright 2015 Florian Schmaus
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smack;
+
+import org.igniterealtime.smack.inttest.AbstractSmackIntegrationTest;
+import org.igniterealtime.smack.inttest.SmackIntegrationTest;
+import org.igniterealtime.smack.inttest.SmackIntegrationTestEnvironment;
+import org.jivesoftware.smack.chat.Chat;
+import org.jivesoftware.smack.chat.ChatManager;
+import org.jivesoftware.smack.chat.ChatManagerListener;
+import org.junit.Assert;
+
+
+/**
+ * @author Stawicki Adam
+ */
+public class ChatManagerListenerTest extends AbstractSmackIntegrationTest {
+    
+    private final ChatManager chatManagerOne;
+    
+    private boolean invoked;
+
+    public ChatManagerListenerTest(SmackIntegrationTestEnvironment environment) {
+        super(environment);
+        chatManagerOne = ChatManager.getInstanceFor(conOne);
+        chatManagerOne.addChatListener(new TestChatManagerListener());
+    }
+    
+    @SmackIntegrationTest
+    public void testccc() {
+        chatManagerOne.createChat(conTwo.getUser());
+        Assert.assertTrue("TestChatManagerListener wasn't invoked", invoked);
+    }
+    
+    public class TestChatManagerListener implements ChatManagerListener {
+
+        @Override
+        public void chatCreated(Chat chat, boolean createdLocally) {
+            System.out.println("creating!!!!!!!!!\n\n\n\n\n");
```

Comment:
never use println in a library or unit tests. but checkstyle already told you so.